src/main/resources/static/js/subscription.js

```diff
@@ -495,6 +492,31 @@ jQuery(document).ready(function () {
         table.responsive.recalc();
     });
```

Comment:
Note that although this pr consists of more lines of codes, the intent is to make it more readable and easier to debug.